src/js/components/CheckBox/CheckBox.js

```diff
@@ -191,8 +192,14 @@ const CheckBox = forwardRef(
             if (onChange) onChange(event);
           }}
         />
-        {visual}
-        {hidden}
+        {children ? (
+          children({ checked })
```

Comment:
might the callee want `indeterminate` as well?

Reply:
okay, I will add it.